ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Suggest adding cell about this and giving customers some information about what you're creating here. Recommendation:</p><p> </p><p>"In the following cell, you generate an input manifest file. This file tells Ground Truth where 3D point cloud input data is located and contains metadata about your input data. To learn more, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-point-cloud-input-manifest.html" target="_blank">Create an Input Manifest File for a 3D Point Cloud Labeling Job</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Updated it</p>